### PR TITLE
Bug 2018443 - On a DU node deployed via ZTP process the performance-operator pod has limits set

### PR DESCRIPTION
**Pull request by dgonyier**
Fix to test case for BZ#2018443

### CODE REVIEW

File: functests/1_performance/performance.go [hunk 1]
```diff
@@ -106,9 +106,11 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 			// If workload partitioning is enabled on the node, resources.limits.cpu index is mutated to
 			// resources.limits."management.workload.openshift.io/cores" index
 
-			if wp_cpu_limit, ok := pod.Spec.Containers[0].Resources.Limits["management.workload.openshift.io/cores"]; ok {
-				By("Workload Partitioning enabled, checking Resources.Limits[\"management.workload.openshift.io/cores\"]")
-				Expect(wp_cpu_limit.IsZero()).To(BeTrue(), "Container has \"management.workload.openshift.io/cores\" Limit != 0")
+			if _, ok := pod.Spec.Containers[0].Resources.Limits["management.workload.openshift.io/cores"]; ok {
+				// Ignore limits set for Workload Partition mutation of CPU requests (custom resource)
+				// https://bugzilla.redhat.com/show_bug.cgi?id=2019924
```
Comment (fromanirh) [inline functests/1_performance/performance.go:111]: then just
```
if _, ok := pod.Spec.Containers[0].Resources.Limits["management.workload.openshift.io/cores"]; !ok {
    // Ignore limits set for Workload Partition mutation of CPU requests (custom resource)
    // https://bugzilla.redhat.com/show_bug.cgi?id=2019924
    // https://bugzilla.redhat.com/show_bug.cgi?id=2018443
   By("Checking Resources.Limits.Cpu()")
   Expect(pod.Spec.Containers[0].Resources.Limits.Cpu().IsZero()).To(BeTrue(),
   ...
```

**Review by fromanirh — COMMENTED** [review]
I think you should squash the first two commits together and then rebase it on top of master. No merge commits on PRs, please.

**Comment by cynepco3hahue** [comment]
@dgonyier Please check @fromanirh comment and we can merge it.

File: functests/1_performance/performance.go (hunk 1 above)
Comment (dgonyier) [reply functests/1_performance/performance.go:111]: Fixed in commit  a1fb31b

**Comment by dgonyier** [comment]
> I think you should squash the first two commits together and then rebase it on top of master. No merge commits on PRs, please.

This should be cleaned up now

**Review by fromanirh — COMMENTED** [review]
/lgtm
/hold
next time please squash the fixes in the commit they are fixing

**Review by fromanirh — COMMENTED** [review]
/lgtm

**Pull request merged**